recipes-connectivity/wifi/wifi-systemd-service.bb

```diff
@@ -0,0 +1,37 @@
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7f619bab123dad"
+
+RDEPENDS_${PN} += " kernel-modules linux-firmware-rpidistro-bcm43455 wpa-supplicant "
+
+SRC_URI = "\
+    file://26-dhcp-wireless.network \
+    file://wpa_supplicant-wlan0.conf \
+    "
+
+do_install () {
+
+    if [ -z ${RPI_WIFI_SSID} ]; then
+        bbfatal "RPI_WIFI_SSID is not defined !"
+    fi
+    if [ -z ${RPI_WIFI_PWD} ]; then
+        bbfatal "RPI_WIFI_PWD is not defined !"
+    fi
+    install -d ${D}/${systemd_unitdir}/network
+    install -m 0644 ${WORKDIR}/26-dhcp-wireless.network ${D}/${systemd_unitdir}/network
```

Comment:
It's preferred to use `${sysconfdir}` instead of hardcoding `etc`.

Reply:
agree